src/pages/signup/index.js

```diff
@@ -47,82 +48,126 @@ const Signup = () => {
     handleSubmit,
     formState: { errors },
   } = useForm({ resolver: zodResolver(schema) });
-  const onSubmit = data => signup(data);
+  const [isSend, setIsSend] = useState(false);
+  const [toSend, setToSend] = useState({
+    to_name: '',
+    message: `Hello, thank you for signing up to Target MVD!`,
+    to: '',
+  });
```

Comment:
WDYA defining this message object outside the component and reference it directly in the `useState` hook? It'll make the component cleaner in my opinion, but it's obviously just a matter of preference